Intersect.Network/Intersect.Network.csproj

```diff
@@ -19,12 +19,12 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Lidgren.Network\Lidgren.Network\Lidgren.Network.csproj" />
```

Comment:
So, instead of a package we are going to use this project now, right?
this one has been referenced properly but the project is nowhere to be found right now, not sure if you missed it / this is in progress right now (?) regards 👍🏽